dv_utils/client.py

```diff
@@ -59,7 +113,7 @@ def write_results(self, user_id: str, filename: str, content: str):
             raise Exception("Unsupported result file name: " + filename)
 
         self.request(
-            f"/clients/{self.settings.client_id}/applications/{self.settings.app_id}/activeUsers/{user_id}/{filename}",
+            f"/clients/{self.settings.collaboration_space_owner_id}/applications/{self.settings.collaboration_space_id}/activeUsers/{user_id}/{filename}",
```

Comment:
I don't think this endpoint exists (anymore). I can't find it back in the code.
We should use `/collaborationSpaces/{space_id}/users/{user_id}/appData/{path}` (this checks if the user gave permission)